docker-start.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+[ -z "$UID" ] && UID=0
+[ -z "$GID" ] && GID=0
+
+# echo >> /etc/xxx and not adduser/addgroup because adduser/addgroup won't work if uid/gid already exists
+echo -e "xuser:x:${UID}:${GID}:xuser:/home/xuser:/bin/ash\n" >> /etc/passwd
```

Comment:
`/home/xuser` won't exist, which can cause issues. I think we can just set the home directory `/app` which is guaranteed to exist.